examples/web/assets/js/app.js

```diff
@@ -43,6 +43,9 @@ desktopJS.Electron.ElectronContainer.prototype.showNotification = function (titl
 document.addEventListener("DOMContentLoaded", function (event) {
 	hostName.innerHTML = container.hostType + " &#8226; " + container.uuid + " &#8226; "+ desktopJS.version;
 
+	$("#button-joingroup").prop("disabled", !container.getCurrentWindow().allowGrouping);
+	$("#button-leavegroup").prop("disabled", !container.getCurrentWindow().allowGrouping);
```

Comment:
jQuery was added for twitter bootstrap styling so I'm taking advantage of this now.  I will be doing a future PR to change all of app.js to use jQuery to simplify things.